Fix resource selection during campaign copy

29vivek opened the pull request:
Clear the selection of resource (template, page, profile) whenever the original resource is deleted and there is
only one currently available resource present in the DB while copying a campaign. Without this fix, the only
available resource is shown as the original resource, instead of showing [Deleted].

glennzw commented:
Great catch @29vivek, I can confirm I see the issue on my side and that your fix works. Please could you remove the `.min.js` version as we like to create those on our side - and I'll get this merged in!

Thank you for your patience with us getting to this.

29vivek committed:
Fix resource selection during campaign copy

Clear the selection of resource (template, page, profile) whenever the original
resource is deleted and there is only one currently available resource present
in the DB while copying a campaign. Without this fix, the only available
resource is shown as the original resource, instead of showing [Deleted].

29vivek commented:
Hi @glennzw, I've removed the `.min.js` version. Thanks!
